Gemfile

```diff
@@ -1,40 +1,5 @@
 source 'https://rubygems.org'
-
```

Comment:
@isangieri Why are you changing our gem file in this commit?  Is that related to the bug?